bin/broker_submission.py

```diff
@@ -20,28 +20,40 @@
 from ebi_eva_common_pyutils.logger import logging_config as log_cfg
 
 from eva_submission.eload_brokering import EloadBrokering
+from eva_submission.eload_utils import check_existing_project
 from eva_submission.submission_config import load_config
 
 logger = log_cfg.get_logger(__name__)
 
 
+def ENA_Project(project):
+    # Helper class to validate early that the project provided exist in ENA and is public
```

Comment:
This is a helper function rather than a class... and the name should also be lower-cased.

Reply:
Fixed the comment now. The reason the function is upper cased is because it is used in the argparse error message:
`broker_submission.py: error: argument --project_accession: invalid ENA_Project value: 'PRJ'` 

This also explain why the value error does not have any message (and I've logged it instead) because the message is ignored by argparse.

It is not very elegant from the code point of view but seems to be better from an usability point of view.

Reply:
Thanks for explaining, makes sense now!